Card/card.css

```diff
@@ -0,0 +1,117 @@
+.card {
+  width: 20rem;
+
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.card-header {
+  position: relative;
+}
+
+.card-shadow {
+  border-radius: 2px;
+  box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.5);
+}
+.card-image {
+  border-radius: 3px 3px 0 0;
+  width: 100%;
+  height: auto;
+}
+.card-title {
+  font-size: 1.2rem;
+  font-weight: 600;
+}
+.card-subtitle {
+  font-size: 1.2rem;
+}
+.btn {
+  margin: 0;
+}
+
+.card-footer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+}
+.card-button {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+.card-icons {
```

Comment:
**FEAT:** Here you can give these icons a "cursor: pointer;" as doing so will help the user know that it's a clickable item. With the default cursor, one cannot understand whether it's a button or just an image.